api/taskcluster/webhook.go

```diff
@@ -227,11 +291,11 @@ func parseTaskclusterURL(targetURL string) (rootURL, taskGroupID, taskID string)
 	return rootURL, taskGroupID, taskID
 }
 
-func getTaskGroupInfo(rootURL string, groupID string) (*taskGroupInfo, error) {
+func (api *apiImpl) GetTaskGroupInfo(rootURL string, groupID string) (*TaskGroupInfo, error) {
```

Comment:
Pointer is not needed here

Reply:
I assume you mean a pointer receiver is not needed here (versus the pointer return type)? Fixed.

Reply:
Sorry about the confusion. Right a pointer receiver is not needed. You also need to remove the `&` [here](https://github.com/web-platform-tests/wpt.fyi/pull/1970/commits/1575cc0f3b8cd3a98fed89fc40766ae139fa0e5b#diff-5ed03f025b533061a381c9d5a256ae2cR148)

Reply:
Sorry, I totally missed this comment before clicking the merge button... but somehow the code builds? 

@Hexcles do you know why it seems to be ok to pass a `&apiImpl{}` to a function which requires `api API`?

Reply:
https://play.golang.org/p/vhzr2Qz2Fgk. I think pointers are dereferenced automatically in this case, but it doesn't work other way around (https://play.golang.org/p/_yKicUCK730)

Reply:
Because `API` is an interface, which simply defines some function signatures; and your `apiImpl` now has no function that takes a pointer type as the receiver, so both the `apiImpl` struct and a pointer to it have all the functions attached and hence satisfy the `API` interface, whereas in your original implementation with `(api *apiImpl) GetTaskGroupInfo`, the struct `apiImpl{}` does NOT have `GetTaskGroupInfo` so it can't be used as `API`.